user/e16/template.py

```diff
@@ -0,0 +1,61 @@
+pkgname = "e16"
+pkgver = "1.0.30"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--prefix=/usr",
+    "--sysconfdir=/etc",
+    "--mandir=/usr/share/man",
+    "--localstatedir=/var",
+    "--enable-sound",
+    "--enable-sound-alsa",
+    "--enable-sound-pulse",
+]
+hostmakedepends = ["automake", "libtool", "pkgconf"]
```

Comment:
if it builds with libtool swapped with slibtool, put that there instead

Reply:
I did a rebuilt with `slibtool` and everything looks OK, so I changed it, thanks!